### PR TITLE
ETC: MobName fixes

### PR DESCRIPTION
**Pull request by Sunaries**

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -1020,17 +1020,17 @@ ETC_20150317_001019	Hungry Lauzinute
 ETC_20150317_001020	Mischievous Jukopus
 ETC_20150317_001021	Fast Weaver Foot
 ETC_20150317_001022	Field Chapperition Skill Dummy
-ETC_20150317_001023	Abyss Pawnd
-ETC_20150317_001024	Abyss Rodelin
+ETC_20150317_001023	Abysmal Pond
+ETC_20150317_001024	Abysmal Rodelin
 ETC_20150317_001025	It has been said that the Helgacercle was made by the servants of the demon realm's aristocrats.
-ETC_20150317_001026	Ponderu of Abyss
+ETC_20150317_001026	Abysmal Pondel
```
Comment (ghost) [inline ETC.tsv:1026]: Do you mean Abysmal or Abyssal?

Comment (Sunaries) [inline ETC.tsv:1026]: Abysmal. 

http://blog.al.com/living-press-register/2009/06/bubba_abysmal_vs_abyssal_a_dee.html

Comment (ghost) [inline ETC.tsv:1026]: Alrighty. No issues here. :+1:

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**